lib/opencensus/trace/span_context.rb

```diff
@@ -132,6 +136,16 @@ def trace_options
       #
       attr_reader :span_id
 
+      ##
+      # Whether the parent of spans created by this context is local, or `nil`
+      # if this context creates root spans or this information is unknown.
+      #
+      # @return [boolean, nil]
```

Comment:
Semantically, I feel like any `?` method should only return boolean values.

Reply:
That's a good point. Renamed the method (and the corresponding arguments) to `same_process_as_parent` to emphasize the connection to the field it eventually populates.